### DISCUSSION TITLE
/zh-cn/blog/posts/fuse_api

### DISCUSSION THREAD

**Discussion by giscus** [Reverier-Xu/discussions#141] [Announcements]
# katzekit 2 - FUSE 文件系统 - Reverier's Blog

文件系统是一种用来存储和组织计算机文件的软件。我们日常所说的硬盘只是一种存储介质，可以理解为一张白纸，但是这张纸并不能对上面的数据进行分块管

https://blog.woooo.tech/posts/fuse_api/

**Comment by arttnba3** [comment]
能⑧能讲讲 kernel pwn 怎么在条件竞争里用 FUSE 替代 userfaultfd（

**Reply by Reverier-Xu** [reply]
👴又不会pwn（
